src/courses/beginner/08.md

```diff
@@ -86,14 +84,6 @@ inspec exec my_nginx --reporter json junit2:tmp/junit.xml | tee out.json
 ```
 :::
 
-<!-- How can you list two profiles? That doesn't make sense to me -->
-<!-- ::: tip Flag order
```

Comment:
this could be another detail section that we elide during the class but at least it's there for someone to read on their own w/r to the `--` thing allowing you to move the positional parameter of the profile name to the end of the inspec exec invocation